### PR TITLE
Apply Black to the python examples in the tutorial

### PR DESCRIPTION
**Pull request by peterjc**
This pull request addresses in part issue #2008

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Doc/Tutorial/chapter_cluster.tex [hunk 1]
```diff
@@ -202,11 +202,7 @@ \subsection*{Calculating the distance matrix}
 \end{minted}
 which can be rewritten as
 \begin{minted}{python}
-[array([], dtype=float64),
- array([ 16.]),
- array([ 64.,  16.]),
- array([  1.,   9.,  49.])
-]
+[array([], dtype=float64), array([16.0]), array([64.0, 16.0]), array([1.0, 9.0, 49.0])]
```
Comment (peterjc) [inline Doc/Tutorial/chapter_cluster.tex]: This change is arguably less readable - all in one line for a (triangular) matrix

File: Doc/Tutorial/chapter_cluster.tex [hunk 2]
```diff
@@ -370,20 +366,15 @@ \subsection*{$k$-medoids clustering}
 \begin{itemize}
 \item as a 2D Numerical Python array (in which only the left-lower part of the array will be accessed):
 \begin{minted}{python}
-distance = array([[0.0, 1.1, 2.3],
-                  [1.1, 0.0, 4.5],
-                  [2.3, 4.5, 0.0]])
+distance = array([[0.0, 1.1, 2.3], [1.1, 0.0, 4.5], [2.3, 4.5, 0.0]])
```
Comment (peterjc) [inline Doc/Tutorial/chapter_cluster.tex]: This change (and the similar ones later in this file) is arguably less readable - all in one line for a matrix

File: Doc/Tutorial/chapter_cookbook.tex [hunk 3]
```diff
@@ -1622,13 +1688,33 @@ \subsection{Creating your own substitution matrix from an alignment}
...
+    ("D", "R"): 48.0,
+    ("E", "R"): 2.0,
+    ("D", "K"): 1.0,
+    ("E", "K"): 45.0,
+    ("K", "R"): 130.0,
+    ("E", "D"): 241.0,
+    ("E", "E"): 3305.0,
+    ("D", "E"): 270.0,
+    ("D", "D"): 2360.0,
+}
```
Comment (peterjc) [inline Doc/Tutorial/chapter_cookbook.tex]: This is longer but more readable.

File: Doc/Tutorial/chapter_graphics.tex [hunk 4]
```diff
@@ -593,15 +627,59 @@ \subsection{Multiple tracks}
...
+)
+C_colors = (
+    [grey] * 30
+    + [green] * 5
+    + [brown] * 4
+    + [blue] * 2
+    + [grey, blue]
+    + [lightblue] * 2
+    + [grey] * 5
+)
```
Comment (peterjc) [inline Doc/Tutorial/chapter_graphics.tex]: Cross reference discussion on #2373 which affect the complete copy of this script under ``Doc/examples/``

**Comment by peterjc** [comment]
@svalqui would you like to review this?

File: Doc/Tutorial/chapter_advanced.tex [hunk 5]
```diff
@@ -35,10 +35,9 @@ \section{Substitution Matrices}
...
 \subsection{SubsMat}
 
-This module provides a class and a few routines for generating substitution matrices, similar to BLOSUM or PAM matrices, but based on user-provided data. Additionally, you may select a matrix from MatrixInfo.py, a collection of established substitution matrices. The \verb+SeqMat+ class derives from a dictionary:
-\begin{minted}{python}
-class SeqMat(dict)
-\end{minted}
+This module provides a class and a few routines for generating substitution matrices, similar to BLOSUM or PAM matrices, but based on user-provided data. Additionally, you may select a matrix from MatrixInfo.py, a collection of established substitution matrices.
+
+The \verb+SeqMat+ class derives from a dictionary.
 The dictionary is of the form \verb|{(i1,j1):n1, (i1,j2):n2,...,(ik,jk):nk}| where i, j are alphabet letters, and n is a value.
```
Comment (svalqui) [inline Doc/Tutorial/chapter_advanced.tex]: is the class part removed by black?

Comment (peterjc) [reply Doc/Tutorial/chapter_advanced.tex]: No, not black (or tool for applying black to the documentation). If you look at the commit comments this was deliberate - it was hard to get a meaningful code snippet with valid syntax, and I didn't think it added much.

**Comment by peterjc** [comment]
To clarify the editorial choices, the tool would only fix a file if all the examples were valid syntax and black could be run (all or nothing). Therefore the first commits are fixing syntax, with the last commit actually using the tool ``blacken_docs.py`` to run black on the code snippets.

**Comment by svalqui** [comment]
I couldn't find anything that could be improved on the style. Looks good to me.

**Comment by peterjc** [comment]
Thank you!

**Pull request merged**